StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+API.swift

```diff
@@ -290,6 +317,14 @@ extension PaymentSheet {
 
             switch confirmOption {
             case .wallet:
+                guard let intent = confirmableIntent.asIntent else {
+                    let message =
+                        "Cannot present Link without an Intent"
+                    assertionFailure(message)
+                    completion(.failed(error: PaymentSheetError.unknown(debugDescription: message)))
```

Comment:
does this mean Link doesn't work right now w/ decoupled?

We could `let intent = confirmableIntent.asIntent ?? await fetchIntent(..)` here for now and leave it as a future optimization to remove the Payment/SetupIntent object dependency from PayWithLinkController.

Reply:
oof yes and no, it works in complete mode but not flow controller, going to fix this and add tests.